### ISSUE TITLE
Server appears as offline

### ISSUE THREAD

**Issue by ghost** [OvercastNetwork/OCN#8]
Hello, I configured all the OCN website as the readme file says, but I can't connect the API trough the OCN website, and when I start the custom Bungee of OCN, appears this
`16:49:59 [INFORMACIÓN] [MinecraftServiceImpl] Connected to API as server.58cc59c2125c3726af000002`
I started up rails octc, rails api, and config/worker.rb, and I don't have any plugin at Bungee Cord (Only API and Commons Bungee), look, there is an screenshot

![foto1](https://cloud.githubusercontent.com/assets/6548993/24064800/b3324b7e-0b34-11e7-8735-dbc044d26752.png)

**Comment by iPolish** [comment]
You have vía version plugin

**Comment by ghost** [comment]
No, that was an old Screenshot, now, I don't have any plugin, look
![screen](https://cloud.githubusercontent.com/assets/6548993/24073197/9c168bfc-0bc1-11e7-925c-1201976f1276.png)

**Comment by chemistryx** [comment]
@ianfe9 You need API-OCN
